src/generic/_generic.ms-custom-properties.scss

```diff
@@ -9,7 +9,9 @@
 :root {
     @each $scale in map-get($FLUIDMS-CONFIG, scales) {
         --ms#{$scale}-ratio:       #{fluidms-pow(map-get($FLUIDMS-CONFIG, ratio), $scale)};
-        --ms#{$scale}-line-height: #{fluidms-line-height($scale, map-get($FLUIDMS-CONFIG, ratio))};
+        @if $FLUIDMS-GLOBAL-BASELINE {
+            --ms#{$scale}-line-height: #{fluidms-line-height($scale, map-get($FLUIDMS-CONFIG, ratio))};
```

Comment:
why does the baseline have a unit (`rem`) when applied to `line-height`?

Reply:
The baseline in general needs to have a unit, because it is not tied to any font-size. As we want it all to be fluid, the unit needs to be `rem`. 

Since we are managing the line-heights via CSS custom properties, we cannot use unit-less line-heights, because ... well ... it’s very complicated to calculate (if even possible), especially when using the `$line-height-modifier` argument. But more importantly, when a global baseline grid is needed, I see no problem in specifically defining the line-heights in `rem`s, as the baseline unit itself is defined in `rem`s, too. Don’t forget, that there can’t be one single unit-less line-height **for all font-sizes** anyway, when the global baseline grid is active. 

It’s simpler and more clear this way.